tests/direct_scraper/test_direct_scraper.py

```diff
@@ -23,7 +26,7 @@ def tearDown(self):
         mozfile.rmtree(self.temp_dir)
 
     def test_url_download(self):
-        test_url = 'https://mozqa.com/index.html'
+        test_url = urljoin(self.wdir, 'download_test.txt')
```

Comment:
The problem why this is not working is that `setUp()` is not calling itself via the base class. But given that the code for creating the tmp folder is already present in `MozHttpdBaseTest.setUp` you can get rid of the whole method. The same applies also for `tearDown`.